test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Create.cshtml

```diff
@@ -1,10 +1,7 @@
-﻿@model ValidationWebSite.Models.Person
+﻿@model BasicWebSite.Models.RemoteAttributeUser
 @{
-    object areaObject;
-    ViewContext.ActionDescriptor.RouteValueDefaults.TryGetValue("area", out areaObject);
-    var areaName = (areaObject as string) ?? "root";
-
-    ViewBag.Title = "Create in " + areaName + " area.";
+    Layout = "/Areas/Area1/Views/RemoteAttribute_Home/_Layout.cshtml";
+    ViewBag.Title = "Create in Area1 area.";
```

Comment:
Separately just use a relative path for the `Layout` -- `Layout = "_Layout.cshtml";`.